app/services/provider_authorisation.rb

```diff
@@ -3,6 +3,13 @@ def initialize(actor:)
     @actor = actor
   end
 
+  def can_manage_users?
```

Comment:
There is no universal `can_manage_users?` permission, the ability to manage users is granted on a per-provider basis.

Something like 
```suggestion
  def can_manage_users_for?(provider)
```
would be more accurate.

See also https://github.com/DFE-Digital/apply-for-teacher-training/pull/2530/files#diff-6c36ce09043ee29972a80d5a8862669dR154

and https://github.com/DFE-Digital/apply-for-teacher-training/pull/2530/files#diff-6c36ce09043ee29972a80d5a8862669dR141

Reply:
I think it's just named confusingly, not flawed. Its purpose is to check whether or not the user has permission to manage users on at least one org so we can show the page in the navigation. @duncanjbrown had the same comment in https://github.com/DFE-Digital/apply-for-teacher-training/pull/2543#discussion_r458228462. I've renamed the method in https://github.com/DFE-Digital/apply-for-teacher-training/commit/553111bebab0b4bcebaf6074d2f77767676bbca0.

Reply:
Great, thanks